chains/orchestrator-relays/runtime/starlight/constants/src/lib.rs

```diff
@@ -0,0 +1,190 @@
+            pub EthereumLocation: Location = Location::new(1, EthereumNetwork::get());
+
+    }
+
+    #[cfg(feature = "runtime-benchmarks")]
+    parameter_types! {
+            // We need a different ethereum location for benchmarks as the ethereum system pallet
+            // is written for benchmarks from para
+            pub EthereumLocationForParaIdBenchmarks: Location = Location::new(2, EthereumNetwork::get());
+
+    }
+}
+
+/// Fee-related.
+pub mod fee {
+    pub use sp_runtime::Perbill;
+    use {
+        crate::weights::ExtrinsicBaseWeight,
+        frame_support::weights::{
+            WeightToFeeCoefficient, WeightToFeeCoefficients, WeightToFeePolynomial,
+        },
+        primitives::Balance,
+        smallvec::smallvec,
+    };
+
+    /// The block saturation level. Fees will be updates based on this value.
+    pub const TARGET_BLOCK_FULLNESS: Perbill = Perbill::from_percent(25);
+
+    /// Handles converting a weight scalar to a fee value, based on the scale and granularity of the
+    /// node's balance type.
+    ///
+    /// This should typically create a mapping between the following ranges:
+    ///   - [0, `frame_system::MaximumBlockWeight`]
+    ///   - [Balance::min, Balance::max]
+    ///
+    /// Yet, it can be used for any other sort of change to weight-fee. Some examples being:
+    ///   - Setting it to `0` will essentially disable the weight fee.
+    ///   - Setting it to `1` will cause the literal `#[weight = x]` values to be charged.
+    pub struct WeightToFee;
+    impl WeightToFeePolynomial for WeightToFee {
+        type Balance = Balance;
+        fn polynomial() -> WeightToFeeCoefficients<Self::Balance> {
+            // in Starlight, extrinsic base weight (smallest non-zero weight) is mapped to 1/10 CENT:
+            let p = super::currency::CENTS;
+            let q = 10 * Balance::from(ExtrinsicBaseWeight::get().ref_time());
+            smallvec![WeightToFeeCoefficient {
+                degree: 1,
+                negative: false,
+                coeff_frac: Perbill::from_rational(p % q, q),
+                coeff_integer: p / q,
+            }]
+        }
+    }
+}
+
+/// System Parachains.
+pub mod system_parachain {
+    use {primitives::Id, xcm_builder::IsChildSystemParachain};
+
+    /// Network's Asset Hub parachain ID.
+    pub const ASSET_HUB_ID: u32 = 1000;
+    /// Contracts parachain ID.
+    pub const CONTRACTS_ID: u32 = 1002;
+    /// Encointer parachain ID.
+    pub const ENCOINTER_ID: u32 = 1003;
+    /// People parachain ID.
+    pub const PEOPLE_ID: u32 = 1004;
+    /// BridgeHub parachain ID.
+    pub const BRIDGE_HUB_ID: u32 = 1013;
+    /// Brokerage parachain ID.
+    pub const BROKER_ID: u32 = 1005;
+
+    /// All system parachains of Starlight.
+    pub type SystemParachains = IsChildSystemParachain<Id>;
+}
+
+/// Starlight Treasury pallet instance.
+pub const TREASURY_PALLET_ID: u8 = 40;
```

Comment:
is this true? is it instance 40?

Reply:
Yes, it is:
<img width="1201" alt="Zrzut ekranu 2025-04-23 o 13 44 12" src="https://github.com/user-attachments/assets/f9682731-a7ea-401b-b75a-3bcecd7883f9" />